### PR TITLE
Modified MAUI Calendar Team Side Control Getting Started Content

### PR DESCRIPTION
**Pull request by Sathish-087**
Modified MAUI calendar team side control getting started content.

- Picker
- Linear Gauge
- StepProgressBar
- Shimmer
- Cards
- Date Picker
- Time Picker

### CODE REVIEW

File: MAUI/Cards/getting-started.md [hunk 1]
```diff
@@ -7,19 +7,73 @@ control: SfCard
...
 
-4. Initialize `SfCards.`
+### Visual Studio Code
+
+1. Open the command palette by pressing `Ctrl+Shift+P` and type **.NET:New Project** and enter.
+2. Choose the **.NET MAUI App** template.
+3. Select the project location, type the project name and press enter.
+4. Then choose **Create project.**
+
+## Step 2: Install the Syncfusion MAUI ProgressBar NuGet Package
```
Comment (MuniappanSubramanian) [inline MAUI/Cards/getting-started.md:34]: mentioned it as progress bar- change it to cards

File: MAUI/Picker/getting-started.md [hunk 2]
```diff
@@ -7,51 +7,39 @@ control: SfPicker
...
-    . . .
-    xmlns:picker="clr-namespace:Syncfusion.Maui.Picker;assembly=Syncfusion.Maui.Picker">
+1. Open the command palette by pressing `Ctrl+Shift+P` and type **.NET:New Project** and enter.
+2. Choose the **.NET MAUI App** template.
+3. Select the project location, type the project name and press enter.
+4. Then choose **Create project.**
 
-    <picker:SfPicker />
-</ContentPage>
+## Step 2: Install the Syncfusion MAUI ProgressBar NuGet Package
```
Comment (MuniappanSubramanian) [inline MAUI/Picker/getting-started.md:34]: change control name to picker

Comment (Sathish-087) [reply MAUI/Picker/getting-started.md]: Changed control name

File: MAUI/Cards/getting-started.md (hunk 1 above)
Comment (Sathish-087) [reply MAUI/Cards/getting-started.md]: Changed control name

**Review by MuniappanSubramanian — APPROVED** [review]
This is fine

File: MAUI/Cards/getting-started.md [hunk 3]
```diff
@@ -7,19 +7,76 @@ control: SfCard
 documentation: ug
 ---
 
-# Getting started with .NET MAUI Card (SfCard)
-This section details the process of integrating the Card control and focuses solely on the fundamental features required for initiating your exploration of Syncfusion Card.
+# Getting started with .NET MAUI Card
+This section details the process of integrating the [.NET MAUI Cards](https://www.syncfusion.com/maui-controls/maui-cards) control and focuses solely on the fundamental features required for initiating your exploration of Syncfusion Card. Follow the steps below to add a basic cards view to your project.
```
Comment (Jeyasri-Murugan) [inline MAUI/Cards/getting-started.md:11]: Follow the steps below to add .NET MAUI Cards control to your project

File: MAUI/Cards/getting-started.md [hunk 4]
```diff
@@ -7,19 +7,76 @@ control: SfCard
...
+3. Select the .NET framework version and click **Create**.
+
+### Visual Studio Code
+
+1. Open the command palette by pressing `Ctrl+Shift+P` and type **.NET:New Project** and enter.
+2. Choose the **.NET MAUI App** template.
+3. Select the project location, type the project name and press enter.
+4. Then choose **Create project.**
+
+## Step 2: Install the Syncfusion MAUI Cards NuGet Package
```
Comment (Jeyasri-Murugan) [inline MAUI/Cards/getting-started.md:38]: Install the Syncfusion .NET MAUI Cards NuGet Package

File: MAUI/Cards/getting-started.md [hunk 5]
```diff
@@ -7,19 +7,76 @@ control: SfCard
...
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Step 4: Add a Basic cards view.
```
Comment (Jeyasri-Murugan) [inline MAUI/Cards/getting-started.md:76]: Add .NET MAUI Cards control

File: MAUI/Cards/getting-started.md [hunk 6]
```diff
@@ -7,19 +7,76 @@ control: SfCard
...
-1. Create a new .NET MAUI application in Visual Studio.
+## Prerequisites
 
-2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfCards` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Cards](), and then install.
+Before proceeding, ensure the following are set up:
+1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later is installed.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here.](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code)
 
-3. To initialize the control, import the control namespace `Syncfusion.Maui.Cards` in XAML or C# code.
+## Step 1: Create a New MAUI Project
```
Comment (Jeyasri-Murugan) [inline MAUI/Cards/getting-started.md:23]: Create a New .NET MAUI Project

File: MAUI/Cards/getting-started.md [hunk 7]
```diff
@@ -7,19 +7,76 @@ control: SfCard
...
+
+## Step 2: Install the Syncfusion MAUI Cards NuGet Package
+
+1. In **Solution Explorer,** right-click the project and choose **Manage NuGet Packages.**
+2. Search for [Syncfusion.Maui.Cards](https://www.nuget.org/packages/Syncfusion.Maui.Cards/) and install the latest version.
+3. Ensure the necessary dependencies are installed correctly, and the project is restored.
+
+## Step 3: Register the handler
+
+The [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core/) NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.
```
Comment (Jeyasri-Murugan) [inline MAUI/Cards/getting-started.md:46]: highlight it - MauiProgram.cs

**Pull request merged**